format/format.go

```diff
@@ -24,9 +27,13 @@ type Host struct {
 }
 
 // PrettyPrintJSON output indented json via stdout.
```

Comment:
`func PrettyPrintJSON` may not output indented json if filtered value is single.